packages/react-jss/src/injectSheet.test.js

```diff
@@ -37,6 +37,21 @@ describe('injectSheet', () => {
       })()
     })
 
+    it.skip('should work in StrictMode without error on React 16.3+', () => {
```

Comment:
I really like the test, but why are you skipping it?

Reply:
Was commenting it out for the one leftover componentWillReceiveProps, but removing it because it seems that I can just remove the componentWillReceiveProps.